process_manager.py

```diff
@@ -0,0 +1,34 @@
+import signal
+import time
+from enum import Enum
+
+SystemState = {
+  'ACTIVE' : 'ACTIVE',
+  'PAUSED' : 'PAUSED',
+  'SHUTDOWN' : 'SHUTDOWN'
+}
```

Comment:
`SystemState` is a dict, maybe should be an Enum as you hinted in the import?

Reply:
I made it a dict to allow more descriptive list of commands over MQTT rather than numbers, and then easily just index from the dict by the msg string over mqtt